packages/eas-cli/src/commands/update/configure.ts

```diff
@@ -54,69 +55,55 @@ export default class UpdateConfigure extends EasCommand {
       await installExpoUpdatesAsync(projectDir);
     }
 
-    const [androidWorkflow, iosWorkflow] = await Promise.all([
-      resolveWorkflowAsync(projectDir, Platform.ANDROID),
-      resolveWorkflowAsync(projectDir, Platform.IOS),
-    ]);
+    const workflows = await resolveWorkflowPerPlatformAsync(projectDir);
```

Comment:
optional: can put all three of these methods into a single utility method to ensure they are kept consistent between the two commands
```typescript
function configureAsync(...) {
  const workflows = await resolveWorkflowPerPlatformAsync(projectDir);
  const updatedExp = await configureAppJSONForEASUpdateAsync(...);
  await configureNativeFilesForEASUpdateAsync(...)
}
```
and then call this function from both `configure` and `update`.

Reply:
I think that @byCedric plan to refactor the `configure` code anyway, and if this is out of scope of his planned changes I can tackle that in a separate PR.